src/sonic-py-common/tests/device_info_test.py

```diff
@@ -51,9 +53,16 @@
 
 
 class TestDeviceInfo(object):
-    @classmethod
-    def setup_class(cls):
-        print("SETUP")
+    @pytest.fixture(scope="class", autouse=True)
+    def sanitize_environment(self):
+        # Back up environment variables then clear them, in case an environment
```

Comment:
>Back up [](http://example.com/codeflow?start=10&length=7)

Mocking is a better way to create an isolated environment.
ref: [https://adamj.eu/tech/2020/10/13/how-to-mock-environment-variables-with-pytest/](https://adamj.eu/tech/2020/10/13/how-to-mock-environment-variables-with-pytest/)

Reply:
Refactored to use `mock` in latest commit.